### PR TITLE
Additional weapons fallback data

### PR DESCRIPTION
**Pull request by Nosajthedevil**
Added weapons fallback data for the laser guided rockets, AIM-54's, AGM-65 E/F, 4X Hellfire and TOW carried by US helis, rockets for them to fall back to, all of the USSR IR missiles, R-3R, Mk-83 & Mk-84 and adjusted the dates for the 
R-27ER &ET to 1990.

This is my first time creating a pull request and my first contribution to liberation, let me know if there are any issues with what and how I'm submitting.

Will likely work more of these over the next few weeks if I get time -

Thanks

### CODE REVIEW

File: resources/weapons/a2a-missiles/K-13A.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+name: K-13A
+year: 1962
+fallback:
```
Comment (DanAlbert) [inline resources/weapons/a2a-missiles/K-13A.yaml]: If there is no fallback the field should be omitted.

File: resources/weapons/a2a-missiles/RS2US.yaml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+name: RS2US - AAM, beam-rider
+year: 1957
+fallback:
```
Comment (DanAlbert) [inline resources/weapons/a2a-missiles/RS2US.yaml]: And here

File: resources/weapons/bombs/Mk-83.yaml [hunk 3]
```diff
@@ -1,16 +1,17 @@
...
-  - "{Mk_83P}"
-  - "{Mk_83PT}"
-  - "{BRU42_MK83 RS}"
-  - "{BRU3242_MK83 RS}"
-  - "{PHXBRU3242_MK83 RS}"
-  - "{7A44FF09-527C-4B7E-B42B-3F111CFE50FB}"
-  - "{BRU42_MK83 LS}"
-  - "{BRU3242_MK83 LS}"
-  - "{PHXBRU3242_MK83 LS}"
+name: Mk 83
```
Comment (DanAlbert) [inline resources/weapons/bombs/Mk-83.yaml]: It looks like your git client isn't configured with https://docs.github.com/en/get-started/getting-started-with-git/configuring-git-to-handle-line-endings#global-settings-for-line-endings. I just added a `.gitattributes` file to enforce it. After rebasing your changes onto develop you'll need to run `git add --renormalize .` to fix these.

(why does this matter? it makes the diff harder to read if every line in the file changes. afaict the thing that changed here is that a fallback was added, but it's hard to tell)

File: resources/weapons/rockets/AGR-20 M282.yaml [hunk 4]
```diff
@@ -0,0 +1,5 @@
+name: LAU-131 pod - 7 x 2.75\" Hydra, Laser Guided Rkts M282, MPP APKWS
+year: 2012
+fallback: AGM-65F
```
Comment (DanAlbert) [inline resources/weapons/rockets/AGR-20 M282.yaml]: I wonder if this should fall back to the E/L maverick, and maybe to a double or triple rack? Hard to say about the second given the huge weight difference.

File: resources/weapons/rockets/AGR-20 M282.yaml [hunk 5]
```diff
@@ -0,0 +1,5 @@
+name: LAU-131 pod - 7 x 2.75\" Hydra, Laser Guided Rkts M282, MPP APKWS
```
Comment (DanAlbert) [inline resources/weapons/rockets/AGR-20 M282.yaml]: These don't have to match the in game name (which is the reason for the clsid mapping; the game names change somewhat frequently). You can name them something simpler like `LAU-131 7xMPP APKWS` if you prefer.

**Review by DanAlbert — CHANGES_REQUESTED** [review]
Mostly looks good. Left a few notes.

File: resources/weapons/rockets/AGR-20 M282.yaml (hunk 5 above)
Comment (Nosajthedevil) [reply resources/weapons/rockets/AGR-20 M282.yaml]: I did not know that. That is a lot easier, thanks.

File: resources/weapons/bombs/Mk-83.yaml (hunk 3 above)
Comment (Nosajthedevil) [reply resources/weapons/bombs/Mk-83.yaml]: You are correct - I added a fallback to this and the Mk-84. I didn't use a standalone client. I may have worked all of this incorrectly. I created a fork in Github, created or edited yaml files, uploaded them to the fork, then issued a pull request. I'll spend some time figuring out git and, going forward, will work any further changes through an editor

File: resources/weapons/rockets/AGR-20 M282.yaml (hunk 4 above)
Comment (Nosajthedevil) [reply resources/weapons/rockets/AGR-20 M282.yaml]: That makes sense. The E/L maverick will be included in the next batch, probably early next week when I get time. I can then update this to fallback to it or whatever else makes sense.

**Review by Nosajthedevil — COMMENTED** [review]
Thanks for your patience on this. This is all very new to me, still trying to figure it out.

File: resources/weapons/a2a-missiles/K-13A.yaml (hunk 1 above)
Comment (Nosajthedevil) [reply resources/weapons/a2a-missiles/K-13A.yaml]: Done

File: resources/weapons/a2a-missiles/RS2US.yaml (hunk 2 above)
Comment (Nosajthedevil) [reply resources/weapons/a2a-missiles/RS2US.yaml]: Done

File: resources/weapons/bombs/Mk-83.yaml (hunk 3 above)
Comment (DanAlbert) [reply resources/weapons/bombs/Mk-83.yaml]: If you don't figure it out I can fix it for you before you submit easily enough. After you update your branch with he new changes in develop it'll be automatic on your end, but only for the files you touch from then on.

File: resources/weapons/rockets/AGR-20 M282.yaml (hunk 4 above)
Comment (DanAlbert) [reply resources/weapons/rockets/AGR-20 M282.yaml]: We can also merge this as-is and get the update later.

Comment (Nosajthedevil) [reply resources/weapons/rockets/AGR-20 M282.yaml]: That sounds like a plan. Thank you

**Review by DanAlbert — APPROVED** [review]

**Comment by DanAlbert** [comment]
I rechecked this against the latest develop and it all seems fine. Sorry for the lack of response here. I distinctly remember _writing_ a message saying that I would look at this in time for 6.0 but wanted to recheck first, but I guess I forgot to hit send 😞

**Comment by Nosajthedevil** [comment]
No worries, I actually started mapping out the Russian air to ground weapons but never got around to finishing it.  Wound up using time period specific loadouts for the su24 and su17 instead. Will revisit sometime soon

**Comment by DanAlbert** [comment]
Sounds good. I'm sure the work will still be here when you find the time 😁

**Pull request merged**